### PR TITLE
make sure that the value escaped is a string

### PR DESCRIPTION
**Pull request by agroszer**

### CODE REVIEW

File: src/z3c/table/column.py [hunk 1]
```diff
@@ -71,6 +71,13 @@ def addColumn(
     return column
 
 
+def htmlEscape(value):
+    # make sure that the value escaped is a string
+    # otherwise html.escape would burp on any non-string
+    # the result will be anyway converted to a string with % interpolation
+    return html.escape("%s" % value)
```
Comment (alga) [inline src/z3c/table/column.py]: Why not `html.escape(str(value))`?

Comment (agroszer) [reply src/z3c/table/column.py]: actually, why not `u"%s" % value`

Comment (kedder) [reply src/z3c/table/column.py]: `UnicodeError` on this line in all cases.

Comment (agroszer) [reply src/z3c/table/column.py]: okay, we figured we'll fix our code

**Pull request closed without merging**